app/gateway/changelog.md

```diff
@@ -753,6 +753,50 @@ This should be a 400 because the configuration is invalid.
 * When the OpenID Connect (OIDC) plugin is configured to reference HashiCorp Vault in the `config.client_secret` field (for example, `{vault://hcv/clientSecret}`),
 it does not look up the secret correctly.
 
+## 3.2.2.4
+**Release Date** 2023/08/21
+
+### Enterprise
+
+#### Breaking Changes
+
+- Ubuntu 18.04 artifacts are no longer supported as they are end-of-life.
+- Amazon Linux 2022 artifacts are renamed to Amazon Linux 2023, based on AWS's own renaming.
+
+#### Deprecations
+
+- **CentOS packages are now removed from the release and are no longer supported in future versions.**
+
+#### Dependencies
+
+* `lua-resty-kafka` is bumped from 0.15 to 0.16
+
+#### Fixes
```

Comment:
❗ For reviewers: When I looked at the issues linked to some of these fixes, they had multiple pull requests associated with them, so I didn't add a link to any PR. Is that correct? Also, many of them had fix versions that weren't "3.2.2.4", is this something I need to ask about or can we assume these are also in 3.2.2.4 since they were in the changelog draft?

Reply:
If the linked PRs are in a private repo (usually Kong/kong-ee), then we don't link to them.

If they're links to a public repo, you can have more than one PR link if it's relevant. However, a lot of the PRs in a linked list will be '"backported/cherry picked to X", which we don't link to. So use your best judgement :)